# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/alessandro/vlan/multi-line-string-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- vlan/vconfig.c.orig	2004-03-27 10:52:29.000000000 +0100
+++ vlan/vconfig.c	2004-03-27 10:53:35.000000000 +0100
@@ -37,31 +37,30 @@
 
 
 static char* usage = 
-      "
-Usage: add             [interface-name] [vlan_id]
-       rem             [vlan-name]
-       set_flag        [interface-name] [flag-num]       [0 | 1]
-       set_egress_map  [vlan-name]      [skb_priority]   [vlan_qos]
-       set_ingress_map [vlan-name]      [skb_priority]   [vlan_qos]
-       set_name_type   [name-type]
-
-* The [interface-name] is the name of the ethernet card that hosts
-  the VLAN you are talking about.
-* The vlan_id is the identifier (0-4095) of the VLAN you are operating on.
-* skb_priority is the priority in the socket buffer (sk_buff).
-* vlan_qos is the 3 bit priority in the VLAN header
-* name-type:  VLAN_PLUS_VID (vlan0005), VLAN_PLUS_VID_NO_PAD (vlan5),
-              DEV_PLUS_VID (eth0.0005), DEV_PLUS_VID_NO_PAD (eth0.5)
-* bind-type:  PER_DEVICE  # Allows vlan 5 on eth0 and eth1 to be unique.
-              PER_KERNEL  # Forces vlan 5 to be unique across all devices.
-* FLAGS:  1 REORDER_HDR  When this is set, the VLAN device will move the
-            ethernet header around to make it look exactly like a real
-            ethernet device.  This may help programs such as DHCPd which
-            read the raw ethernet packet and make assumptions about the
-            location of bytes.  If you don't need it, don't turn it on, because
-            there will be at least a small performance degradation.  Default
-            is OFF.
-";
+"\n"
+"Usage: add             [interface-name] [vlan_id]\n"
+"       rem             [vlan-name]\n"
+"       set_flag        [interface-name] [flag-num]       [0 | 1]\n"
+"       set_egress_map  [vlan-name]      [skb_priority]   [vlan_qos]\n"
+"       set_ingress_map [vlan-name]      [skb_priority]   [vlan_qos]\n"
+"       set_name_type   [name-type]\n"
+"\n"
+"* The [interface-name] is the name of the ethernet card that hosts\n"
+"  the VLAN you are talking about.\n"
+"* The vlan_id is the identifier (0-4095) of the VLAN you are operating on.\n"
+"* skb_priority is the priority in the socket buffer (sk_buff).\n"
+"* vlan_qos is the 3 bit priority in the VLAN header\n"
+"* name-type:  VLAN_PLUS_VID (vlan0005), VLAN_PLUS_VID_NO_PAD (vlan5),\n"
+"              DEV_PLUS_VID (eth0.0005), DEV_PLUS_VID_NO_PAD (eth0.5)\n"
+"* bind-type:  PER_DEVICE  # Allows vlan 5 on eth0 and eth1 to be unique.\n"
+"              PER_KERNEL  # Forces vlan 5 to be unique across all devices.\n"
+"* FLAGS:  1 REORDER_HDR  When this is set, the VLAN device will move the\n"
+"            ethernet header around to make it look exactly like a real\n"
+"            ethernet device.  This may help programs such as DHCPd which\n"
+"            read the raw ethernet packet and make assumptions about the\n"
+"            location of bytes.  If you don't need it, don't turn it on, because\n"
+"            there will be at least a small performance degradation.  Default\n"
+"            is OFF.\n";
 
 void show_usage() {
    fprintf(stdout,usage);
